backstop.config.js

```diff
@@ -0,0 +1,56 @@
+const { urls, baseUrl } = require('./urls');
+
+const createScenariosFromUrls = (urls, delay = 1000) => {
+  return Object.keys(urls).map((label) => ({
+    label,
+    url: urls[label],
+    selectors: ['document'],
+    misMatchThreshold: 0.3,
+    requireSameDimensions: true,
+    delay,
+  }));
+};
+
+module.exports = {
+  id: 'backstop_test',
+  viewports: [
+    {
+      label: 'desktop',
+      width: 1024,
+      height: 768,
+    },
+  ],
+  scenarios: [
```

Comment:
Very cool!  Is this the part of the config that we'd add more paths to check to have tested in the future?